dashboard/db/migrate/20180525181121_create_teacher_feedbacks.rb

```diff
@@ -0,0 +1,12 @@
+class CreateTeacherFeedbacks < ActiveRecord::Migration[5.0]
+  def change
+    create_table :teacher_feedbacks do |t|
+      t.text :comment
+      t.integer :student_id
+      t.integer :level_id
+      t.integer :teacher_id
+
+      t.timestamps
+    end
```

Comment:
What about indexes? What will be the common queries? If we're, say, typically returning all feedbacks for a given student, we should index `student_id`

Reply:
I think our most common query will be when students look up their feedback on a level (which is looked up by student_id and level_id: https://github.com/code-dot-org/code-dot-org/pull/22789/files#diff-85c0b816fb78a84b91283bdda9798782R13) - it'll happen when a student clicks to "Feedback" tab on a level. The look-up for teachers might also be pretty popular (whenever a teacher tries to add/update/look-at feedback for one of their students): https://github.com/code-dot-org/code-dot-org/pull/22789/files#diff-85c0b816fb78a84b91283bdda9798782R6 

I'll add indices for both of these situations.